prettify_components.sh

```diff
@@ -0,0 +1,18 @@
+# Add this as a shell script to your WebStorm configurations
+# and as executable command to VS Code
+# Make sure the working directory for this command is equal to "<PROJECT_DIR>/src/components"
+
+# The path to the prettier config file.
+CONFIG_FILE="../../.pretierrc"
```

Comment:
`pretierrc` → `prettierrc`?

Also, personally I prefer to name that file `.prettierrc.json` with the `.json` extension so it has the correct syntax highlighting and is formatted itself by Prettier too, and any accidental syntax errors can be picked up by linting.

Reply:
In my IDE it already shows as a JSON file. Likely a `.prettier`-file already translates to JSON in the background as it read as such. Also, syntax highlighting seems to work fine for me.
![afbeelding](https://user-images.githubusercontent.com/43957930/115961444-a28dd600-a516-11eb-9aa1-a366a4d2836e.png)